__init__.py

```diff
@@ -15,7 +15,50 @@ class MarkExperience(Event):
 
 
 MODIFIER_REGEX = "[+-]?[0,1,2,3]"
-STAT_REGEX = f"(?:(?:!cool|!tough|!sharp|!charm|!weird) {MODIFIER_REGEX})"
+GAME_STATS = {"motw": ["cool", "tough", "sharp", "charm", "weird"],
+              "pbtastartrek": ["aggressive", "bold", "talk", "tech,", "morale", "shields"]}
+STAT_REGEXES = {}
+
+
+@match_event(OpsdroidStarted)
+async def migrate_old_keys(opsdroid, config, event):
+    db = opsdroid.get_database("matrix")
+    for room in opsdroid.get_connector('matrix').connection.rooms.values():
+        with db.memory_in_room(room.room_id):
+            old_key = await opsdroid.memory.get("motw_stats") or {}
```

Comment:
Why the or?
```suggestion
            old_key = await opsdroid.memory.get("motw_stats")
```

Reply:
Because a I didn't mean that and b forgot that what I was trying to do isn't necessary anyway.